gravitee-apim-rest-api/gravitee-apim-rest-api-portal/gravitee-apim-rest-api-portal-rest/src/main/java/io/gravitee/rest/api/portal/rest/resource/SubscriptionsResource.java

```diff
@@ -79,6 +80,8 @@ public class SubscriptionsResource extends AbstractResource {
     @Inject
     private KeyMapper keyMapper;
 
+    private static final ObjectMapper MAPPER = new ObjectMapper();
```

Comment:
Is it possible to use the `GraviteeMapper` instead ?

Reply:
:heavy_check_mark: